src/CMakeLists.txt

```diff
@@ -252,31 +204,12 @@ endif()
 
 target_link_libraries(parthenon PUBLIC Kokkos::kokkos)
 
-lint_target(parthenon)
-
 target_include_directories(parthenon PUBLIC
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/generated>
+  $<BUILD_INTERFACE:${parthenon_SOURCE_DIR}/include>
+  $<BUILD_INTERFACE:${parthenon_BINARY_DIR}/include/parthenon/generated>
   $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/parthenon>
```

Comment:
I think something needs to be fixed here - either the install still needs to put the headers into a `parthenon` directory, or this just needs to be `${CMAKE_INSTALL_INCLUDEDIR}`.

Reply:
Good catch.